app/controllers/profile/niphl/UpdateIsNiphlRegisteredController.scala

```diff
@@ -33,14 +33,13 @@ import views.html.profile.HasNiphlView
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
-class HasNiphlController @Inject() (
+class UpdateIsNiphlRegisteredController @Inject() (
```

Comment:
split into 2 controllers one for update one for create. renamed HasNiphl to IsNiphlRegistered as the is nearer to the content on the page and seems a better match.  Page content "Are you NIPHL registered?"